sdk/tests/TransactionReceiptQueryTest.cc

```diff
@@ -59,3 +65,34 @@ TEST_F(TransactionReceiptQueryTest, SetTransactionId)
   query.setTransactionId(transactionId);
   EXPECT_EQ(query.getTransactionId(), transactionId);
 }
+
+// Tests invoking of method execute() from TransactionReceiptQuery.
+TEST_F(TransactionReceiptQueryTest, ExecuteRequestToTestnetNode)
+{
+  // Given
+  const auto testPublicKey = ED25519PrivateKey::generatePrivateKey()->getPublicKey();
+  const auto accountOperatorId = AccountId::fromString("0.0.1359");
+  Client testnetClient = Client::forTestnet();
+  testnetClient.setOperator(
+    accountOperatorId,
+    ED25519PrivateKey::fromString("2e000363977258a41f418cf84a7df9cf1e8ae98b72de86803e64846defa43054"));
+
+  AccountCreateTransaction testAccountCreateTransaction;
+  testAccountCreateTransaction.setKey(testPublicKey);
+
+  TransactionId testTransactionId =
+    testAccountCreateTransaction.execute(testnetClient).getRecord(testnetClient).getTransactionId().value();
+
+  TransactionReceiptQuery testTransactionReceiptQuery;
+  testTransactionReceiptQuery.setTransactionId(testTransactionId);
+  testTransactionReceiptQuery.setMaxAttempts(3);
```

Comment:
What is the purpose of setting this here?

Reply:
You're correct - calling of `setMaxAttempts(3)` could be removed here.

Reply:
Fixed in ee729d39770a80f56917cc8f0921fab2f6882e75.